src/module/migration/migrations/810-loot-description-value.ts

```diff
@@ -0,0 +1,20 @@
+import { ActorSourcePF2e } from "@actor/data";
+import { MigrationBase } from "../base";
+
+/** Remove value property from loot actor description */
+export class Migration810LootDescriptionValue extends MigrationBase {
+    static override version = 0.81;
+
+    override async updateActor(source: ActorSourcePF2e): Promise<void> {
+        if (source.type === "loot") {
+            const details: MaybeWithValue = source.system.details;
+            if (typeof details.description === "object") {
```

Comment:
use `instanceof Object` just in case of the extremely unlikely scenario that `description` is `null`